examples/RenderManagerD3DATWDoubleBufferExample.cpp

```diff
@@ -39,6 +39,7 @@ Russ Taylor working through ReliaSolve.com for Sensics, Inc.
 #include <iostream>
 #include <string>
 #include <chrono>
+#include <mutex>
```

Comment:
This include would only be needed if you were using one of the `std::mutex` variants, which you aren't (at least in this diff)